admin_guide/scheduling/taints_tolerations.adoc

```diff
@@ -0,0 +1,398 @@
++
+----
+admissionConfig:
+  pluginConfig:
+    DefaultTolerationSeconds:
+      configuration:
+        kind: DefaultAdmissionConfig
+        apiVersion: v1
+        disable: false
+----
+
+. Restart OpenShift for the changes to take effect:
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+----
+endif::[]
+
+. Verify that the default was added:
++
+.. Create a pod:
++
+----
+$ oc create -f </path/to/file>
+----
++
+For example:
++
+----
+$ oc create -f hello-pod.yaml
+pod "hello-pod" created
+----
++
+.. Check the pod tolerations:
++
+----
+$ oc describe pod <pod-name> |grep -i toleration
+----
++
+For example:
++
+----
+$ oc describe pod hello-pod |grep -i toleration
+Tolerations:    node.alpha.kubernetes.io/notReady=:Exists:NoExecute for 300s
+----
+
+[[admin-guide-taints-daemonsets]]
+== Daemonsets and Tolerations
+
+link:https://kubernetes.io/docs/admin/daemons/[DaemonSet] pods are created with `NoExecute` tolerations for `node.alpha.kubernetes.io/unreachable` and `node.alpha.kubernetes.io/notReady`
+with no `tolerationSeconds` to ensure that DaemonSet pods are never evicted due to these problems, even when the Default Toleration Seconds feature is disabled.
+
+[[admin-guide-taints-use-cases]]
+== Example Use Cases
+
+Taints and tolerations are a flexible way to steer pods away from nodes or evict pods that should not be running on a node. A few of the use cases are:
+
+* xref:admin-guide-taints-dedicated-user[Dedicating a node for a user]
+* xref:admin-guide-taints-dedicated-only[Binding a user to a node]
+* xref:admin-guide-taints-special[Dedicating nodes with special hardware]
+
+[[admin-guide-taints-dedicated-user]]
+=== Dedicating a Node for a User
+
+You can specify a set of nodes for exclusive use by a particular set of users.
+
+To specify dedicated nodes:
+
+. Add a taint to those nodes:
++
+For example:
++
+----
+$ oadm taint nodes node1 dedicated=groupName:NoSchedule
+----
+
+. Add a corresponding toleration to the pods. This would be done most easily by writing a custom xref:../install_config/master_node_configuration.adoc#master-config-admission-control-config[admission controller].
++
+Only the pods with the tolerations will be allowed to use the dedicated nodes.
+
+[[admin-guide-taints-dedicated-only]]
+=== Binding a User to a Node
+
+You can configure a node so that particular users can use only the dedicated nodes.
+
+To configure a node so that users can use only that node:
+
+. Add a taint to those nodes:
++
+For example:
++
+----
+$ oadm taint nodes node1 dedicated=groupName:NoSchedule
+----
+
+. Add a corresponding toleration to the pods. This would be done most easily by writing a custom xref:../install_config/master_node_configuration.adoc#master-config-admission-control-config[admission controller].
```

Comment:
To avoid the "easily" adverb, maybe:

Add a corresponding toleration to the pods by writing a custom admission controller."